src/index.ts

```diff
@@ -4,20 +4,32 @@ import { exec } from "child_process";
 import { join } from "path";
 
 export default function run(commandArgs: string[]) {
-  const strykerConfPath = join(process.cwd(), "stryker.conf.js");
+
+  const args = commandArgs.slice(2);
+  let branch;
+  let path;
+
+  mapAllArgumentsToAnArray(args).forEach((element) => {
+    switch (element[0]) {
+      case "branch":
+        branch = element[1];
+        break;
+      case "path":
+        path = element[1];
+    }
+  });
```

Comment:
Something like :

```suggestion
branch = mapAllArgumentsToAnArray(args).find(argArray => argArray[0] === "branch");
branch = mapAllArgumentsToAnArray(args).find(argArray => argArray[0] === "path");
```

What do you think ?

We could also declare `branch` and `path` as `const`.